### PR TITLE
fix(DB/SAI): Lashh'an Wing Guard

### PR DESCRIPTION
**Pull request by avarishd**
## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/4954
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/15127

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

https://www.wowhead.com/wotlk/npc=19944/lashhan-wing-guard
Source for GO position is mangos-wotlk DB

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

.go c id 19944
Bring it to 20% hp
Should run to nearest (out of 2) runes and summon bird

## Known Issues and TODO List:

- [ ] https://github.com/azerothcore/azerothcore-wotlk/issues/19171

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Gultask** [comment]
after https://github.com/azerothcore/azerothcore-wotlk/pull/19190

he angy no move
image

made a condition for u :)
```
DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 22) AND (`SourceGroup` = 2) AND (`SourceEntry` = 19944) AND (`SourceId` = 0) AND (`ElseGroup` = 0) AND (`ConditionTypeOrReference` = 30) AND (`ConditionTarget` = 1) AND (`ConditionValue1` = 184826) AND (`ConditionValue2` = 100) AND (`ConditionValue3` = 0);
INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
(22, 2, 19944, 0, 0, 30, 1, 184826, 100, 0, 0, 0, 0, '', 'Only execute SAI if Object Lashh\'an Circle Spell Focus is within 100y');
```

**Comment by avarishd** [comment]
I had a condition before, but wanted to give attention to the move to pos bug. Thanks nontheless!
(Running the #19190 now (without condition applied) and they at least no longer run to africa, condition however is indeed required.)

File: data/sql/updates/pending_db_world/rev_1719344429715330100.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+(19944, 0, 0, 0, 0, 0, 100, 0, 3500, 4000, 10500, 14500, 0, 0, 11, 37577, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Lashh\'an Wing Guard - In Combat - Cast \'Debilitating Strike\''),
+(19944, 0, 1, 0, 2, 0, 100, 0, 0, 20, 0, 0, 0, 0, 80, 1994400, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Lashh\'an Wing Guard - Between 0-20% Health - Run Script'),
+(19944, 0, 2, 0, 34, 0, 100, 0, 8, 0, 0, 0, 0, 0, 80, 1994401, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Lashh\'an Wing Guard - On Reached Point 0 - Run Script'),
+(19944, 0, 3, 0, 17, 0, 100, 0, 0, 0, 0, 0, 0, 0, 49, 0, 0, 0, 0, 0, 0, 5, 0, 0, 0, 0, 0, 0, 0, 0, 'Lashh\'an Wing Guard - On Summoned Unit - Start Attacking'),
+(19944, 0, 4, 0, 25, 0, 100, 0, 0, 0, 0, 0, 0, 0, 8, 2, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Lashh\'an Wing Guard - On Reset - Set Reactstate Aggressive');
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 9 AND `entryorguid` IN(1994400,1994401));
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1719344429715330100.sql:14]:
```suggestion
DELETE FROM `smart_scripts` WHERE (`source_type` = 9 AND `entryorguid` IN (1994400,1994401));
```

**Review by Gultask — APPROVED** [review]

**Comment by avarishd** [comment]
I just got PTSD-ed that this was another "-".
